src/rgw/rgw_json_enc.cc

```diff
@@ -427,6 +427,7 @@ void RGWUserInfo::dump(Formatter *f) const
   encode_json("bucket_quota", bucket_quota, f);
   encode_json("user_quota", user_quota, f);
   encode_json("temp_url_keys", temp_url_keys, f);
+  encode_json("type", type, f);
```

Comment:
@pritha-srivastava maybe json encode this enum as a string, not as an int (e.g., look at rgw_meta_sync_info)